numpy/core/arrayprint.py

```diff
@@ -28,12 +35,14 @@
     except ImportError:
         from dummy_thread import get_ident
 
+import numpy as np
```

Comment:
I assumed it was OK since `fromnumeric.py` does it too, but I agree `from .._globals import _NoValue` seems less risky.